source/AfterburnerUsage.cpp

```diff
@@ -0,0 +1,68 @@
+/* AfterburnerUsage.cpp
+Copyright (c) 2022 by Hurleveur
+
+Endless Sky is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free Software
+Foundation, either version 3 of the License, or (at your option) any later version.
+
+Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+this program. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "AfterburnerUsage.h"
+
+
+
+AfterburnerUsage::AfterburnerUsage(const Outfit &outfit) :
+	afterburner(outfit), baseDuration(outfit.Attributes().Get("afterburner duration")),
+	baseCooldown(outfit.Attributes().Get("afterburner cooldown"))
```

Comment:
Pretty sure the durations aren't updated if you uninstall/install afterburners?

Reply:
Oh that isnt like buying then.... I'll look into it

Reply:
AddOutfit isnt used for that? that would be weird

Reply:
Are you sure you did? The only place you add to the afterburner list is in Ship::FinishLoading.

Reply:
in AddOutfit too?